src/main/scala/com/gu/contentapi/services/Ophan.scala

```diff
@@ -9,10 +9,12 @@ object Ophan {
   private val client = new OkHttpClient()
   private val OphanUrl = "https://ophan.theguardian.com/i.gif"
 
+  private val eventsPerSecond = 30
```

Comment:
How do you get this number? My math is terrible, but assuming 3655 worst cast, that means the minimum rate should be 3655 over five minutes, i.e. 3655 / 300 ≃ 12. I may have misunderstood though 😅

Reply:
Ophan currently handles 40000 pageviews per minute (~700 per second) on a [normal day](https://dashboard.ophan.co.uk/graph):

![image](https://user-images.githubusercontent.com/52038/42506007-7cf98486-8438-11e8-8e3e-a733678e79f3.png)

The proposed 30/sec podcast pageviews in this PR are still a low fraction (~5%) of the traffic we see normally, so I think this is fine :+1:

Reply:
@rtyley thanks!